### PR TITLE
Cloud casa doc

### PR DESCRIPTION
**Pull request by chris-milsted**
First commit of the CloudCasa documentation

### CODE REVIEW

**Comment by chris-milsted** [comment]
Merging as the link failures will be resolved once the PR has merged as the images are also in the PR. 
Doc has been reviewed also by CloudCasa and has been given a LGTM via email.

**Review by cannischan — APPROVED** [review]

**Pull request merged**
